src/pip/_internal/index.py

```diff
@@ -511,7 +511,7 @@ def _sort_key(self, candidate):
         """
         valid_tags = self._supported_tags
         support_num = len(valid_tags)
-        build_tag = tuple()  # type: BuildTag
+        build_tag = ()  # type: BuildTag
```

Comment:
Unrelated change?

Reply:
This was caught via `check-builtin-literals`.